js/ui/drop_down_editor/ui.drop_down_list.js

```diff
@@ -873,10 +881,12 @@ const DropDownList = DropDownEditor.inherit({
             case 'items':
                 if(!this.option('dataSource')) {
                     this._processDataSourceChanging();
+                    this._setAriaLabelToList();
```

Comment:
let's move it to `_processDataSourceChanging` to reduce copypaste

Reply:
Good idea! Was fixed in [d55def719df90757723ca998b13c81a9f9a52e0b].